addons/io_hubs_addon/components/ui.py

```diff
@@ -50,6 +50,15 @@ def draw_component(panel, context, obj, row, component_item):
 
         top_row.label(text=display_name)
 
+        if has_properties:
```

Comment:
I think scene components can be skipped here since there's no current support for them?

Reply:
Actually I'm going to poll on the operator to disable in the unsupported cases better